react/training-project/src/app/components/Board/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+
+import Square from '../Square';
+
+import style from './styles.scss';
+
+class Board extends Component {
+  renderSquare = () => <Square />;
+  render() {
+    const status = 'Next player: X';
+    return (
+      <React.Fragment>
+        <div className={style.status}>{status}</div>
+        <div className={style.boardRow}>
+          {this.renderSquare(0)}
```

Comment:
Why are you sending a number as argument if the method doesn't have parameters? :thinking: